backend/app/services/channels/model_selection.py

```
@@ -0,0 +1,146 @@
+            user_id: Wegent user ID
+
+        Returns:
+            ModelSelection object or None if not set
+        """
+        key = ModelSelectionManager._generate_key(user_id)
+        data = await cache_manager.get(key)
+
+        if data:
+            try:
+                return ModelSelection.from_dict(data)
+            except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)